Makefile

```diff
@@ -1,7 +1,8 @@
 APP_NAME = reconciler
 IMG_NAME := $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME)
 TAG := $(DOCKER_TAG)
-COMPONENTS := $(shell recons=$$(find pkg/reconciler/instances -d 1 -type d -not -path '*/example' -execdir echo -n '{} ' \; | xargs) && echo $${recons// /,})
+# COMPONENTS := $(shell recons=$$(find pkg/reconciler/instances -d 1 -type d -not -path '*/example' -execdir echo -n '{} ' \; | xargs) && echo $${recons// /,})
```

Comment:
Reason for changing this was that the `find` command was failing, because `find` has different standard in GNU shell (in debian, prow-job pod). Also, the substitution for `recons` in echo command was also failing in GNU.

The error was:
```
find: paths must precede expression: `1'
/bin/sh: 1: Bad substitution
```